website/static/js/folderPickerNodeConfig.js

```diff
@@ -509,7 +509,20 @@ var FolderPickerViewModel = oop.defclass({
                     custom : FolderPicker.selectView
                 }
             ];
-            }
+            },
+            xhrconfig: $osf.setXHRAuthorization,
+            lazyLoadPreprocess: function(data) {
+                // Also handle data from API -- squash `attributes` to what TB expects
+                // TODO: [OSF-6384] DRY this up when PR #5240 goes in
+                if (data.data) {
+                    $.each(data.data, function(i, obj) {
+                        var saved_attributes = obj.attributes;
```

Comment:
Minor: As per COSDev, use lowerCamelCase for variable names.

I can fix this when I merge.